src/pages/deriv-go/components/_tabs.tsx

```diff
@@ -133,18 +124,22 @@ const Mobile = styled(Show.Mobile)`
     }
 `
 
-const TabPanel = ({ children, className }) => (
-    <TabContent className={className} role="tabpanel" tabindex="0">
+type TabsType = {
+    children: ReactElement | ReactElement[]
+    label?: ReactElement
+    description?: ReactElement
+    className?: string
+    is_reverse?: boolean
+    max_width?: string
+}
+
+const TabPanel = ({ children, className }: TabsType) => (
```

Comment:
```suggestion
const TabPanel = ({ children, className }: TabPanelPropsType) => (
```
I suggest adding a type for TabPanel props as it's a separate component like so 🙂:
type TabPanelPropsType = Pick<TabsType, "children" | "className">;

Reply:
Good point @maryia-binary 🙇 Implemented by your suggestion